src/components/layout/entry-point-wrapper.tsx

```diff
@@ -0,0 +1,51 @@
+"use client";
+
+import { usePathname, useSearchParams } from "next/navigation";
+import { BackButton } from "@/components/layout/back-button";
+import { HelpfulBox } from "@/components/layout/helpful-box";
+import { cn } from "@/lib/utils";
+
+// Without back button; py-8 lg:py-16 (padding-top: 32px; padding-bottom: 32px; @media (min-width: 1024px) { padding-top: 64px; padding-bottom: 64px; })
+// With back button; pt-4 pb-8 lg:py-8 (padding-top: 16px; padding-bottom: 32px; @media (min-width: 1024px) { padding-top: 32px; padding-bottom: 32px; })
+
+// biome-ignore lint/style/useConsistentTypeDefinitions: Do not need `type` features
+interface EntryPointWrapperProps {
+  children: React.ReactNode;
+}
```

Comment:
```suggestion
type EntryPointWrapperProps = {
  children: React.ReactNode;
}
```
Let's use type for consistency through the codebase